template/lambda-sqs-worker/serverless.yml

```diff
@@ -59,6 +59,7 @@ functions:
     reservedConcurrency: 20
     timeout: 30
     environment:
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: 1
```

Comment:
I'm on the fence as to whether it's worth even mentioning but in a similar vein but in a much less important way we could set `AWS_STS_REGIONAL_ENDPOINTS=regional` https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html

Reply:
Ooh awesome! It would be nice to include this and also have links to the relevant doco for both environment variables, as part of documenting more of the rationale behind these templates

Reply:
@tobyhei any reasoning behind not adding that?

Reply:
Oh sorry, I was just going to suggest we add  `AWS_STS_REGIONAL_ENDPOINTS=regional` but then thought it was scope creep for this PR so removed the comment